uni/lib/model/entities/restaurant.dart

```diff
@@ -16,7 +20,11 @@ class Restaurant {
   }
 
   List<Meal> getMealsOfDay(DayOfWeek dayOfWeek) {
-    return meals[dayOfWeek]!;
+    final List<Meal>? meal = meals[dayOfWeek];
+    if(meal == null){
+      return [];
+    }
+    return meal;
```

Comment:
```suggestion
    return meals[dayOfWeek] ?? [];
```

The ?? operator is very useful in these cases where the value can be null and we want another value to be returned in that cases.